app-crypt/acme-dns/acme-dns-0.4.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GOLANG_PKG_IMPORTPATH="github.com/joohoi"
+GOLANG_PKG_TAGS="sqlite3"
```

Comment:
I believe this should be set to `""` since it already uses `+sqlite` by default and by doing we configure it to have `sqlite3` already.

Reply:
@bpinto You're absolutely right, I've changed that and made a fixup into d65f2ba